cli/Cargo.toml

```diff
@@ -11,16 +11,29 @@ readme = "README.md"
 
 [dependencies]
 clap = { version = "4.1.4", default-features = true }
-serde = { version = "1.0.92", features = ["derive"] }
-serde_json = "1.0"
+serde = "1.0.147"
+serde_json = "1.0.87"
+serde_urlencoded = "0.7"
 tokio = { version = "1.25.0", features = ["full"] }
 thiserror = "1.0"
 percent-encoding = "^2.1.0"
+hyper = { version = "0.14", features = ["full"] }
+hyper-rustls = { version = "0.23.1", features =["http2"] }
+anyhow = "1.0.68"
```

Comment:
I wasn't even using anyhow because after I figured out how Results worked, I just used the one that came with Rust and simply forgot to remove the 'use' statements at the beginning of the file